webserver/__init__.py

```diff
@@ -31,6 +33,9 @@ def create_app():
     from kafka_connection import init_kafka_connection
     init_kafka_connection(app.config['KAFKA_CONNECT'])
 
+    # Redis connection
+    app.redis = Redis()
```

Comment:
OK, can we please focus on the other, more important, parts of this PR? I'll fix this, now move on!